docs/development.md

```diff
@@ -61,7 +61,25 @@ sudo apt install python3 python3-pip python3-venv -y
 
    You should see the list of available commands and options.
 
-5. **Run the CLI**
+5. **Run the Automatic Installer**
+
+   Run the python implemented automatic installer using  the
+   install script from the root folder.
+
+   ```bash
+   ./install.sh
```

Comment:
why an `install.sh` script directly in the root?
I would place this under `script` directory.